service_unavailable_page/web/mime.types

```diff
@@ -1,78 +0,0 @@
-types {
```

Comment:
Again; im unsure what this mime types file is doing; something to do with the service unavailable page?

Reply:
The service unavailable page was deployed to the PaaS, and therefore doesn't work on AKS. We probably do want an alternative for the future but I think it's confusing to keep code around which doesn't work, so until then we can remove it.

This particular file was a configuration file for Nginx, to make sure it served the files with the correct type for the browser.